pylint/config/arguments_manager.py

```diff
@@ -392,34 +392,16 @@ def optik_option(
                 optdict.pop(key)
         return args, optdict
 
-    # pylint: disable-next=fixme
-    # TODO: Optparse: All methods below this line are copied to keep API-parity with
-    # OptionsManagerMixIn. They should either be deprecated or moved above this line
-    # to keep them in _ArgumentsManager
-
-    def cb_set_provider_option(self, option, opt, value, parser):
```

Comment:
I have moved these down as option setting is (still) dependent on `3.7` only because we need `re.Pattern`. In order for the `TODO` comment to keep functioning as a "line" I moved them down.